server/api.js

```diff
@@ -3,7 +3,7 @@ import { Router } from "express";
 const router = Router();
 
 router.get("/", (_, res) => {
```

Comment:
Hey Iwona, i see that you are defining home page route(correct me if i am wrong) and i think you forgot to close( ) if you can type 
router.get('/', function (req, res) {
  res.json({ message: "world" });
})